setup.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
```

Comment:
Add short description and example usage here.

Reply:
done, thank you for the feedback. Feel free to improve the script.